.github/dependabot.yaml

```diff
@@ -0,0 +1,16 @@
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    labels: ["area/ci", "dependencies"]
+    schedule:
+      # By default, this will be on a monday.
+      interval: "weekly"
```

Comment:
In the other repos we are setting it as daily https://github.com/weaveworks/pipeline-controller/pull/190/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28R6 

I am not that daily is the right long-term choice (for example due to the amount of PRs raised) but adding it here in case you want to have the same settings as other repos.